content/post/082-our-focus-for-2020.md

```diff
@@ -0,0 +1,86 @@
+
+## Our Focus on Package Managers
+
+Our goal for last year was to improve IPFS performance and scalability by analyzing the needs and pain points for using IPFS in **Package Managers**. This goal was less about _package management_ in particular, and more about defining a representative use case we could research, test, and drive improvements around that would **also** benefit all downstream IPFS users with similar performance and scalability needs for adding, updating, and fetching large datasets. 
+
+Focusing on a representative use case like package managers brought focus and structure to our prioritization of improvements to IPFS. We built a number of proofs-of-concept (POCs) like [apt-on-ipfs](https://github.com/ipfs-shipyard/apt-on-ipfs), [npm-on-ipfs](https://github.com/ipfs-shipyard/npm-on-ipfs), [clojars-on-ipfs](https://github.com/ipfs-shipyard/clojars-mirror-test) and [homebrew-on-ipfs](https://github.com/ipfs/package-managers/issues/12) to analyze how well IPFS performed against user expectations for adding, updating, and fetching large package repositories—allowing us to identify and fix major pain points. For example, our POCs identified a huge bottleneck in adding GBs of data to IPFS. Further testing resulted in _doubling_ the speed of adding on Linux & OSX devices by switching to async-by-default datastores. While IPFS had previously taken 24mins to add an Arch Linux repo, after [these fixes](https://github.com/ipfs/go-ipfs/issues/6523#issuecomment-546822191) it only took 11mins (comparable to the time to copy/paste)! _Note: improvement seen on the badger datastore, already 3x faster than flatfs!_ 
+
+Our Package Managers goal also helped focus our user research, quarterly objectives, and collaborations. In Q1&2, our Package Managers team surveyed the space and identified core needs documented in our [Package Managers repo](https://github.com/ipfs/package-managers). These insights informed our [quarterly OKR planning](https://github.com/ipfs/team-mgmt/blob/master/OKR/PACKAGE-MANAGERS.md), driving features to make it easier to mirror file system package managers on IPFS. One particular addition was adding metadata on “last updated” time to our unixfs data model to support smarter/faster package updates (humbly named “unixfsv1.5” and already implemented in js-ipfs; coming soon to go). A number of these package-manager-focused improvements are all slated to roll out in our next feature release, go-ipfs 0.5.0 - [feel free to follow along here](https://github.com/ipfs/go-ipfs/issues/6776)!
```

Comment:
We might want to make it clear that none of these changes have been released (e.g., you'll need the latest master to get the add speed improvements).

Reply:
added a line for grabbing master to try them out